astropy/io/ascii/tests/data/cds/order/ReadMe

```diff
@@ -0,0 +1,67 @@
+   <Astron. Astrophys. 511, A56 (2010)>
+   =2010A&A...511A..56P
+================================================================================
+ADC_Keywords: Clusters, open ; Stars, giant ; Equivalent widths ; Spectroscopy
+Keywords: stars: abundances - Galaxy: disk -
+          open clusters and associations: general
+
+Abstract:
+    The present number of Galactic open clusters that have high resolution
+    abundance determinations, not only of [Fe/H], but also of other key
+    elements, is largely insufficient to enable a clear modeling of the
+    Galactic disk chemical evolution. To increase the number of Galactic
+    open clusters with high quality measurements, we obtained high
+    resolution (R~30000), high quality (S/N~50-100 per pixel), echelle
+    spectra with the fiber spectrograph FOCES, at Calar Alto, Spain, for
+    three red clump stars in each of five Open Clusters. We used the
+    classical equivalent width analysis method to obtain accurate
+    abundances of sixteen elements: Al, Ba, Ca, Co, Cr, Fe, La, Mg, Na,
+    Nd, Ni, Sc, Si, Ti, V, and Y. We also derived the oxygen abundance
+    using spectral synthesis of the 6300{AA} forbidden line.
+
+Description:
+    Atomic data and equivalent widths for 15 red clump giants in 5 open
+    clusters: Cr 110, NGC 2099, NGC 2420, M 67, NGC 7789.
+
+File Summary:
+--------------------------------------------------------------------------------
+ FileName   Lrecl  Records   Explanations
+--------------------------------------------------------------------------------
+ReadMe         80        .   This file
+table1.dat    103       15   Observing logs and programme stars information
+table5.dat     56     5265   Atomic data and equivalent widths
+--------------------------------------------------------------------------------
+
+See also:
+ J/A+A/455/271 : Abundances of red giants in NGC 6441 (Gratton+, 2006)
+ J/A+A/464/953 : Abundances of red giants in NGC 6441 (Gratton+, 2007)
+ J/A+A/505/117 : Abund. of red giants in 15 globular clusters (Carretta+, 2009)
+
+Byte-by-byte Description of file: table.dat
+--------------------------------------------------------------------------------
+   Bytes Format Units     Label     Explanations
+--------------------------------------------------------------------------------
+   1-  7  A7    ---       Cluster   Cluster name
+   9- 12  I4    ---       Star      
+  14- 20  F7.2  0.1nm     Wave      wave
+                                    ? Wavelength in Angstroms
+  22- 23  A2    ---       El        a
+      24  I1    ---       ion       ?=0
+                                    - Ionization stage (1 for neutral element)
+  26- 30  F5.2  eV        chiEx     [1/20843]?+ Catalogue Identification Number
+  32- 37  F6.2  ---       loggf     ]0/140[?- Temperature class codified (10)
+  39- 43  F5.1  0.1pm     EW        ?=-9.9 Equivalent width (in mA)
+  46- 49  F4.1  0.1pm   e_EW        [1/6]?+ Luminosity class codified (11)
```

Comment:
Can we test one column with a two-character order specifier (other "+=" or "-=")? I feel those are going to be thee most likely ones to get messed up in the regex.